docs/DeveloperGuide.md

```diff
@@ -944,6 +956,8 @@ testers are expected to do more *exploratory* testing.
 1. Initial Launch
 
    1. Download the jar file and copy into an empty folder.
+   
+   1. Ensure there is internet connectivity to utilize NusMods data.
```

Comment:
Should we put this in? It's not required to test the launch and I'm scared if it comes across as too mandatory. We need to remember that requiring internet connection is a tP constraint

Reply:
Ohhh okay make sense! I'll remove it